jwst/extract_2d/extract_2d.py

```diff
@@ -16,7 +16,7 @@
 
 
 log = logging.getLogger(__name__)
-log.setLevel(logging.DEBUG)
+log.setLevel(logging.INFO)
```

Comment:
Actually I believe we need to leave them set to DEBUG level. It was discovered years ago, by Mike D. and myself, that for some reason this setting does not set the default level that will be used whenever the module is executed, but rather behaves more like an upper-limit as to what level of messages are passed through. So if you set it to INFO, debug level messages will not appear, even when the user requests them. If you set it to DEBUG, you only actually see debug level messages when the user requests them. That's why I set all of these values in the various step modules to DEBUG several years ago.